rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -141,6 +141,8 @@ def _load_OOV_params(self) -> None:
     def _check_attribute_vocabulary(self, attribute: Text) -> bool:
         """Check if trained vocabulary exists in attribute's count vectorizer"""
         try:
+            if not hasattr(self.vectorizers[attribute], "vocabulary_"):
+                self.vectorizers[attribute]._validate_vocabulary()
```

Comment:
I would recommend moving this line to the exact place where vectorizers are created, i.e. - inside `_create_independent_vocab_vectorizers` and `_create_shared_vocab_vectorizers`